### ISSUE TITLE
TypeError: multiple values for argument 'cfg_name'

### ISSUE THREAD

**Issue by jakubLangr** [fastai/nbdev#514]
When running nbdev_test_notebook locally and in CI, I get the following error:
```
Traceback (most recent call last):                                                                                                                                                                                 
  File "/home/jakub/miniconda3/envs/e39/bin/nbdev_test_nbs", line 8, in <module>                                                                                                                                   
    sys.exit(nbdev_test_nbs())                                                                                                                                                                                     
  File "/home/jakub/miniconda3/envs/e39/lib/python3.9/site-packages/fastcore/script.py", line 105, in _f                                                                                                           
    tfunc(**merge(args, args_from_prog(func, xtra))) 
  File "/home/jakub/miniconda3/envs/e39/lib/python3.9/site-packages/nbdev/test.py", line 109, in nbdev_test_nbs
    files = nbglob(fname)
  File "/home/jakub/miniconda3/envs/e39/lib/python3.9/site-packages/nbdev/export.py", line 409, in nbglob 
    if recursive == None: recursive=get_config().get('recursive', 'False').lower() == 'true'
  File "/home/jakub/miniconda3/envs/e39/lib/python3.9/site-packages/nbdev/imports.py", line 29, in get_config
    config = Config(cfg_path, cfg_name=cfg_name)
TypeError: __init__() got multiple values for argument 'cfg_name'
```
Given that the entire stacktrace seems to happen outside of my workdir it looks like a bug in nbdev. I have tried `1.1.14` and `1.1.16`. When running around in debugger, it is a bit hard to determine where exactly this comes from as
```
 Config.__file__
*** AttributeError: 'functools._lru_cache_wrapper' object has no attribute '__file__'
```
And there's so many "*" imports, it is tricky to determine where exactly it all comes from.

**Comment by sakinaljana** [comment]
Same here,

For the workaround, you can edit this [file](https://github.com/fastai/nbdev/blob/master/nbdev/imports.py#L29):

From
```python
config = Config(cfg_path, cfg_name=cfg_name)
```

To
```python
config = Config(cfg_name)
```

**Comment by straussmaximilian** [comment]
Hi,
I faced the same issue: CI did stop working because of 
`TypeError: __init__() got multiple values for argument 'cfg_name'`.
I did rerun an old CI run which previously was successful and now stopped working.
After checking the logs, I noticed that there is a new `fastcore` version. In the requirements.txt I had `fastcore` version pinned to 1.3.19. Now after changing the requirements to `fastcore==1.3.21` CI is running through again.
